sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -240,12 +241,10 @@ The available samples are:
 - [Consume events from a known position in the Event Hub partition](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_ConsumeEventsFromAKnownPosition.cs)  
   An example of consuming events, starting at a well-known position in the Event Hub partition.
   
-- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample11_ConsumeEventsByBatch.cs)
-
+- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample11_ConsumeEventsByBatch.cs)  
```

Comment:
These links can be made shorter by using relative links (to this README).
`[Consume events from an Event Hub partition in batches](./samples/Sample11_ConsumeEventsByBatch.cs)`

Reply:
I never remember that actually works.   Thanks!   Given that I'm at the end of my day, I'm going to be conservative this time out and just leave them as-is.   I'll revisit when I move back over to the blob storage checkpointererererer skeleton early next week.